lib/components/PlotContainerSection.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  plotContainerSection: {
+    padding: '4px 0',
+    borderBottom: `1px solid ${theme.palette.grey[300]}`,
+  },
+});
+
+const PlotContainerSection = ({ classes, children }) =>
+  children ? (
```

Comment:
Just display the `children` here and do the checking outside `PlotContainerSection`.